svm/tests/example-programs/transfer-from-account/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "transfer-from-account"
```

Comment:
Just a side note here: I'm wondering if we can make a directory somewhere in the monorepo for "fixture programs". I just had a use case the other day where I needed a simple test program, but I couldn't figure out where to put it, so I went with a builtin processor.

Lmk if you have any ideas!

Reply:
There are a bunch of programs in `programs/sbf`. You can grab one of those. In addition, there is the `example-programs` directory in the SVM.